factory/processing/processComponents.go

```diff
@@ -1268,11 +1268,13 @@ func (pcf *processComponentsFactory) newResolverContainerFactory(
 		return nil, err
 	}
 
+	isSyncedLiteObserver := pcf.config.StoragePruning.ObserverCleanOldEpochsData && !pcf.config.StateTriesConfig.SnapshotsEnabled
```

Comment:
please remove the pcf.config.StoragePruning.ObserverCleanOldEpochsData part. It doesn't matter if it is on or off, all the nodes must have the current epoch data.

Reply:
updated